util/trunc/trunc.go

```diff
@@ -0,0 +1,9 @@
+package trunc
+
+// truncates messages to 40 characters
```

Comment:
I think comment should be `truncates messages to n characters`

Reply:
✔️